.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    "es2015",
+    ["env", {"targets": {"node": "current"}}],
```

Comment:
See: https://babeljs.io/docs/plugins/preset-env/

I need `async` and `await` to run test/screenshot/screenshot.js.